en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -303,6 +303,53 @@ class BreadAPIController extends Controller
+
+From there regular request handling occurs and URL will be check to see if it matches `$allowed_actions` on the [`RequestHandler`](api:SilverStripe\Control\RequestHandler), possibly routed via `$url_handlers`. If an `$allowed_action` i.e. method on the [`RequestHandler`](api:SilverStripe\Control\RequestHandler) is matched, if that method returns a [`RequestHandler`](api:SilverStripe\Control\RequestHandler), then control will now be passed to this nested [`RequestHandler`](api:SilverStripe\Control\RequestHandler) and the URL is again effectively truncated from the left.
+
+### Example of a nested RequestHandler being returned in an action method
+
+- There is yml routing directing the path `one` to `RequestHandlerOne`
+- There is a class `RequestHandlerOne` with `$allowed_actions = ['two']`, and a `two()` method returning `RequestHandlerTwo::create();`
+- There is a class `RequestHandlerTwo` with `$allowed_actions = ['hello']`, and a `hello()` method returning `HTTPResponse::create()->setBody('hello');`
+- Navigating to the URL `/one/two/hello` will return a response with a body of "hello"
+
+### Form, HasRequestHandler, FormRequestHandler, and FormField
+
+[`Form`](api:SilverStripe\Forms\Form) does not extend [`RequestHandler`](api:SilverStripe\Control\RequestHandler), instead it implements the [`HasRequestHandler`](api:SilverStripe\Control\HasRequestHandler) interface that requires a method `getRequestHandler()`. [`Form::getRequestHandler()`](api:SilverStripe\Forms\Form::getRequestHandler()) returns a [`FormRequestHandler`](api:SilverStripe\Forms\FormRequestHandler) which is a subclass of [`RequestHandler`](api:SilverStripe\Control\RequestHandler).
+
+Subclasses of [`RequestHandler`](api:SilverStripe\Control\RequestHandler) and implementors of [`HasRequestHandler`](api:SilverStripe\Control\HasRequestHandler) are treated the same because they will both end up calling `handleRequest()` on the subclass of the appropriate [`RequestHandler`](api:SilverStripe\Control\RequestHandler) subclass.
+
+[`FormRequestHandler`](api:SilverStripe\Forms\FormRequestHandler) includes a `$url_handler` entry `'field/$FieldName!' => 'handleField'` which allows it to handle requests to form fields on the form. [`FormRequestHandler::handleField()`](api:SilverStripe\Forms\FormRequestHandler::handleField()) will find the form field matching `$FieldName` and return it. Control is then passed to the returned form field.
+
+[`FormField`](api:SilverStripe\Forms\FormField) extends [`RequestHandler`](api:SilverStripe\Control\RequestHandler), which means that form fields are able to handle HTTP requests and have the `$allowed_actions` config. This allows form fields have define their own AJAX endpoints.
+
+### Example of an AJAX `FormField` that uses nested `RequestHandler`'s
```

Comment:
```suggestion
### Example of an AJAX `FormField` that uses a nested `RequestHandler`
```

Reply:
Updated to use regular words instead of back tick classnames